downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -97,6 +97,42 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+### Revamped library updates
+
+The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative. 
```

Comment:
Topic says library and this paragraph says connectors, so it is bit confusing. Shall we use consistent workding?

Reply:
Agreed. Updated the title as `Revamped connector updates` with https://github.com/ballerina-platform/ballerina-dev-website/pull/8908/commits/8f18beca8757960898400b0878a0218a3d6441f3.